modelskill/model/dummy.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+from typing import Literal
+
+import pandas as pd
+
+from modelskill.model.point import PointModelResult
+from modelskill.model.track import TrackModelResult
+from modelskill.obs import PointObservation, TrackObservation
+
+
+class DummyModelResult:
+    """Dummy model result that always returns the same value.
+
+    Similar in spirit to <https://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyRegressor.html>
+
+    Parameters
+    ----------
+    data : float, optional
+        The value to return if strategy is 'constant', by default None
+    name : str, optional
+        The name of the model result, by default "dummy"
+    """
```

Comment:
strategy is not documented and please add tiny example also as it is probably not obvious to everyone what this ModelResult is doing :-)